control/control.SMO.xml

```diff
@@ -82,7 +82,7 @@ textdomain="control"
 	<default_patterns>microos-base microos-defaults microos-hardware</default_patterns>
 	Using the Container Host role's list of patterns by default
 -->
-	<default_patterns>microos-base microos-defaults microos-container_runtime microos-hardware</default_patterns>
+	<default_patterns>microos-base microos-defaults microos-container_runtime microos-hardware microos-selinux</default_patterns>
```

Comment:
should not this be dependent on user configuration? What if user disable selinux, does it still makes sense?